playground/vue-legacy/vite.config.ts

```diff
@@ -0,0 +1,35 @@
+import path from 'node:path'
+import fs from 'node:fs'
+import { defineConfig } from 'vite'
+import vuePlugin from '@vitejs/plugin-vue'
+import legacyPlugin from '@vitejs/plugin-legacy'
+
+export default defineConfig({
+  base: '',
+  resolve: {
+    alias: {
+      '@': __dirname
+    }
+  },
+  plugins: [
+    legacyPlugin({
+      targets: ['defaults', 'not IE 11', 'chrome > 48']
+    }),
+    vuePlugin()
+  ],
+  build: {
+    minify: false
+  },
+  // special test only hook
+  // for tests, remove `<script type="module">` tags and remove `nomodule`
+  // attrs so that we run the legacy bundle instead.
+  // @ts-ignore
+  __test__() {
```

Comment:
It seems this `__test__` is not called.
`config` was undefined here and `vite.config.ts` won't be read.
https://github.com/vitejs/vite/blob/05712323b292492e9161a6ff7b20bfce43097dfb/playground/vitestSetup.ts#L252-L270

Reply:
it seems to need to fix it in other PR.